packages/suite/src/utils/suite/env.ts

```diff
@@ -48,13 +46,23 @@ export const getPlatformLanguage = () => {
 };
 
 export const isWeb = () => process.env.SUITE_TYPE === 'web';
+
 export const isDesktop = () => process.env.SUITE_TYPE === 'desktop';
 
+export const getElectronReceiverUrl = async () => {
+    if (isDesktop()) {
+        const addressData = await window.desktopApi!.getServerAddress();
```

Comment:
```suggestion
export const getElectronReceiverUrl = async (pathname: '/buy-post' | '/buy-redirect') => {
   if (window.desktopApi) {
        const serverUrl = await window.desktopApi.getRouteAddress(pathname);
        return serverUrl;
```
looks like you don't have to use (and export) `getServerAddress`

`getRouteAddress` is already there and do exactly what you want